src/styles/main.scss

```diff
@@ -1,3 +1,47 @@
+$block-height: 300px;
+$colors:
+  "1" rgb(100%, 0%, 0%),
+  "2" rgb(80%, 0%, 0%),
+  "3" rgb(60%, 0%, 0%),
+  "4" rgb(40%, 0%, 0%),
+  "5" rgb(20%, 0%, 0%),
+  "6" rgb(0%, 0%, 0%);
+
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-size: 100px;
+}
```

Comment:
move it to normalize or smth else

Reply:
I didn't think it's necessary to create separate files for `html` and `body` in this project)